src/commands/analytics/audience.ts

```diff
@@ -105,8 +105,10 @@ export default class AudienceCommand extends AppCommand {
       promises.push(this.loadLanguagesStatistics(statistics, client, app, startDate, endDate, appVersion));
     }
 
-    if (this.activeUsers) {
+    if (this.activeUsers && appBuild) {
       promises.push(this.loadActiveUsersStatistics(statistics, client, app, startDate, endDate, appVersion, appBuild));
+    } else if (this.activeUsers) {
+      out.text("Please provide app version to get active users statistics.");
```

Comment:
Shouldn't it be a warning instead?

Reply:
`out` only supports `text()` for messages and `help()` for help output.
Other options are: `debug()` (not a part of `out`), which would only show up if debug is enabled (meaning that most customers won't be able to see the message), and `console.log`\`console.warn`, which ignore `--quiet` flags. 
So, `out.text()` is the closest to what we need.